my-app/src/components/login/login.js

```diff
@@ -21,26 +21,41 @@ class Login extends React.Component {
         super(props);
         this.state = {
               username: " ",
-              password: " ", 
+              password: " ",
 
             errors: []
           };
           this.handleSubmit = this.handleSubmit.bind(this);
           this.handlePassChange = this.handlePassChange.bind(this);
           this.handleUserChange = this.handleUserChange.bind(this);
     }
-    
+
   componentDidMount(){
     this.getUsers();
   }
-  
+
   getUsers = _ => {
     fetch('/user')
     .then(response => response.json())
     .then(response => this.setState({users: response.data}))
     .catch(err => console.error(err));
   }
 
+  getUserServer = _ => {
```

Comment:
I ran this and tried to login with an incorrect password, but it just logged me in regardless.

Reply:
I'll look into it

Reply:
For now it should just give an alert and not handle whether or not the the user signs in. The goal was to connect the frontend to the backend and not explicitly handle the response from the backend.

Reply:
I fixed this issue.